[docs] Integrate wptserve documentation (#17039)
web-platform-tests/wpt: docs/writing-tests/python-handlers/index.md

# Python Handlers

Python file handlers are Python files which the server executes in response to
requests made to the corresponding URL. This is hooked up to a route like
`("*", "*.py", python_file_handler)`, meaning that any .py file will be
treated as a handler file (note that this makes it easy to write unsafe
handlers, particularly when running the server in a web-exposed setting).

> The Pyton filess must define a single function `main` with the signature::

Reviewer: ```suggestion
The Python files must define a single function `main` with the signature::
```

# Python Handlers

> Python file handlers are designed to provide a vaguely PHP-like interface

Reviewer: I strongly dislike calling this "vaguely PHP-like".

> where each resource corresponds to a particular python file on the

Reviewer: ```suggestion
where each resource corresponds to a particular Python file on the
```

> filesystem. Typically this is hooked up to a route like ``("*",

Reviewer: Let's not say that's "typically"; for WPT it *is*.

"*.py", python_file_handler)``, meaning that any .py file will be

> treated as a handler file (note that this makes python files unsafe in

Reviewer: ```suggestion
treated as a handler file (note that this makes Python files unsafe in
```

much the same way that .php files are when using PHP).

> Unlike PHP, the python files don't work by outputting text to stdout

Reviewer: Let's not quite so directly reference PHP (which isn't so useful if you don't know what PHP does!), but instead say something more like, "These Python files are assumed to define a single function `main(request, response)`. Note therefore anything written to stdout is not returned as part of the response, as is the case in PHP.
